src/esp/assets/GenericSemanticMeshData.cpp

```diff
@@ -25,11 +25,10 @@ namespace Mn = Magnum;
 namespace esp {
 namespace assets {
 
-std::vector<std::unique_ptr<GenericSemanticMeshData>>
+std::unique_ptr<GenericSemanticMeshData>
```

Comment:
Yes, this is a pre-existing condition, but since you're modifying this part of the code anyway -- here I think the `unique_ptr` wouldn't need to be either.